apps/src/tutorialExplorer/tutorialExplorer.js

```diff
@@ -0,0 +1,374 @@
+import React from 'react';
+
+/**
+ * Entry point to build a bundle containing a set of globals used when displaying
+ * tutorialExplorer.
+ */
+require('babel-polyfill');
+window.React = require('react');
+window.ReactDOM = require('react-dom');
+window.Radium = require('radium');
```

Comment:
Oh, actually that makes sense. I think without that line, webpack will think that someone else has provided you with React and so instead of trying to include a version in your bundle, it assumes it exists already as window.React (which it does not).